### PR TITLE
Update TEST-PAGE-Browser.mdx

### PR DESCRIPTION
**Pull request by mmfred**
## Description

Made some edits to this test page to use for documenting edit processes.

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/docs-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

**Comment by jerelmiller** [comment]
Hey @mmfred would you mind applying these changes to this page as well? https://github.com/newrelic/docs-website/blob/develop/src/manual-edits/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx

This will make sure that these changes stick around between migrating for us. That is much easier to manage as we've already mistakenly deleted and restored this file a couple times 😄

**Comment by jerelmiller** [comment]
(note the `manual-edits` folder)

**Comment by mmfred** [comment]
@jerelmiller - sure. I'm just messing around with some editing workflow instructions. Is the manual-edits folder where we should put changes we want to keep to make sure they're not overwritten when there's a migration pass?

**Comment by jerelmiller** [comment]
> Is the manual-edits folder where we should put changes we want to keep to make sure they're not overwritten when there's a migration pass?

Yep! We copy over everything from that folder after we've migrated and synced with the docs site.

A couple things to note on that `manual-edits` folder:
* The file/folder structure much mimic the `src/content` folder. We copy over the files to their exact location in `src/content`.
* The files inside of `manual-edits` aren't sourced as pages in the site. In order to make sure sure they are, these files need to be copied over to `src/content`. We've got an npm script that will do this for you so that you don't have to do it manually. If you run `yarn copy-edits` in your terminal, this will take all the files in `manual-edits` and move them over to `src/content` so that you can see them in the site. Make sure to commit both files!

**Comment by mmfred** [comment]
> We've got an npm script that will do this for you so that you don't have to do it manually. If you run yarn copy-edits in your terminal, this will take all the files in manual-edits and move them over to src/content so that you can see them in the site.

That's really cool! This PR will be closed after I futz with some instructions. Thanks!

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
@mmfred  you removed the callout, but I strongly feel this info should be in a callout. Please add it back 😿 "

File: src/content/docs/archive-campfire-example.mdx [hunk 1]
```diff
@@ -12,8 +12,6 @@ topics: []
 * Authentication token
 * Campfire room names as applicable
```
Comment (bradleycamacho) [inline src/content/docs/archive-campfire-example.mdx:13]: Hello Michelle!!!

**Pull request closed without merging**